### PR TITLE
Sparkbox sprint 46

### PR DESCRIPTION
**Pull request by rkelbel**
# READY FOR REVIEW

## Summary
* #932 
* #940
* #942 
* #943 

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Run BackstopJS visual regression tests
- [x] Ensure Code Climate passes against the `develop` branch. Link here: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/941
- [x] Assign to Mike for review.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
2. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/templates/components/pattern-alert.html.twig [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{# Override of Decanter's Core Template #}
```
Comment (pookmish) [inline docroot/themes/humsci/humsci_basic/templates/components/pattern-alert.html.twig]: Is there a reason not to make this change in the [hs_layouts module template](https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_layouts/patterns/alert/alert.html.twig)?

File: docroot/themes/humsci/humsci_basic/templates/page.html.twig [hunk 2]
```diff
@@ -3,6 +3,12 @@
 
   {% include humsci_basic ~ '/themes/humsci/humsci_basic/templates/components/brandbar.html.twig' with { modifier_class : brand_bar_variant_classname } %}
 
+  <div class="su-global-message hb-position-relative">
+    {% if page.global_message %}
```
Comment (pookmish) [inline docroot/themes/humsci/humsci_basic/templates/page.html.twig]: this `if` should wrap the div.

**Review by pookmish — CHANGES_REQUESTED** [review]

File: docroot/themes/humsci/humsci_basic/templates/components/pattern-alert.html.twig (hunk 1 above)
Comment (rkelbel) [reply docroot/themes/humsci/humsci_basic/templates/components/pattern-alert.html.twig]: I wasn't sure if you'd want us to alter it there, if it's not going to impact where this is used elsewhere I will make the change in that file and submit a change for it.

File: docroot/themes/humsci/humsci_basic/templates/page.html.twig (hunk 2 above)
Comment (rkelbel) [reply docroot/themes/humsci/humsci_basic/templates/page.html.twig]: Good catch!

**Review by pookmish — APPROVED** [review]

**Pull request merged**